### DISCUSSION TITLE
Timewarp settings help

### DISCUSSION THREAD

**Discussion by sanj3k** [gopro/labs#862] [General]
Hi, I wanted to run a GoPro when riding a longer bike tour in timelapse mode. As far as i have understood it's better to go with timewarp because of the hypersmooth stabilization which makes total sense. What are the remaining tips for better quality and some 180 degree shutter rule etc? Riding a bike should be in the 15-25kph range i don't know if it's an important factor for those settings. Thanks a lot in advance

**Comment by dnewman-gpsw** [comment]
This is not a Labs question.  Maybe ask at https://www.reddit.com/r/gopro/
